controller-server/src/main/java/com/yahoo/vespa/hosted/controller/routing/RoutingPolicies.java

```diff
@@ -133,7 +134,7 @@ public void setRoutingStatus(ZoneId zone, RoutingStatus.Value value) {
                                                                                        controller.clock().instant())));
             Map<ApplicationId, RoutingPolicyList> allPolicies = readAll().groupingBy(policy -> policy.id().owner());
             for (var instancePolicies : allPolicies.values()) {
-                updateGlobalDnsOf(instancePolicies, Set.of(), lock);
+                updateGlobalDnsOf(instancePolicies, Set.of(), Optional.empty(), lock);
```

Comment:
This is the only one I identified as an operator/cross-application event. I'm not even sure we need this distinction, though, because this also happens per application ... so we could probably just do all of these like usual, too? That would simplify a bit.

Reply:
OTOH, this _is_ the right treatment, should any owner-less request appear in the queue, so perhaps leave it like this?

Reply:
(Could alternatively assign an owner to each of the requests generated by taking zones in/out.)

Reply:
Agree that we don't need this. Preserving the order per application is sufficient.